src/utils/styles.js

```diff
@@ -1,16 +1,11 @@
 import autoPrefix from '../styles/auto-prefix';
-import update from 'react-addons-update';
 import warning from 'warning';
 
 const reTranslate = /((^|\s)translate(3d|X)?\()(\-?[\d]+)/;
 
 const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(deg|rad|grad))?/;
 
-function mergeSingle(objA, objB) {
-  if (!objA) return objB;
-  if (!objB) return objA;
-  return update(objA, {$merge: objB});
-}
+export const mergeStyles = (...args) => Object.assign({}, ...args);
```

Comment:
I wish we didn't need this but it was breaking things when we weren't cloning. I think we have some situations in source where we may be passing in an undefined style, but I haven't been able to pinpoint everywhere it's happening yet. I just know calling simply `Object.assign` broke the docs site and the tests.

I think it's worth going back and investigating though! I think the ideal would be that we shouldn't need to do this.